Getting-Started-with-Akka-http-signature.md

```diff
@@ -114,111 +114,77 @@ This will save the contents of the key in String format within the file found by
-  def rename(to: Rdf#Node)(implicit ops: RDFOps[Rdf]): PointedGraph[Rdf] = {
-    val oldNode = pg.pointer
-    PointedGraph[Rdf](
-      to,
-      ops.makeGraph(pg.graph.triples.map{ triple =>
-        ops.fromTriple(triple) match {
-          case (oldNode,rel,obj) => ops.makeTriple(to,rel,obj)
-          case (subj,rel,oldNode) => ops.makeTriple(subj,rel,to)
-          case _ => triple
-        }
-      })
-    )
-  }
-}
-
-defined class PGwrapper
+res44: Try[PointedGraph[Jena]] = Success(org.w3.banana.PointedGraph$$anon$1@5487b93e)
 ```
-
-After this, one can simply change the header by into something simple by calling this function like so:
+The function pubKeyPG returns a Try[PointedGraph[Jena]]. We can then retreive both the pointer and the graph from this PointedGraph like so: 
 
 ```scala
-@ val finalKeyPG = keyPG.rename(URI("#key"))
+@ val keyFinalPointer= keyGraph.get.pointer
 
-finalKeyPG: PointedGraph[Rdf] = org.w3.banana.PointedGraph$$anon$1@4353a749
+keyFinalPointer: Jena#Node = #key
 
-@ finalKeyPGPointer = finalKeyPG.pointer
 
-finalKeyPGPointer : Rdf#Node = #key
+@ val keyFinalGraph = keyGraph.get.graph
 
-@ finalKeyPGGraph = finalKeyPG.graph
+keyFinalGraph: Jena#Graph =  {#key @http://www.w3.org/1999/02/22-rdf-syntax-ns#type http://www.w3.org/ns/auth/cert#RSAPublicKey; #key @http://www.w3.org/ns/auth/cert#modulus "009195c8e386fc421ee8357cc6be674649a9f5722b1dccebab5965a1792c6fafdcc77b9932361f14b5565e5634ec46bfd29f0130aa6d4a2518aed027e7ca34d99a1419262b590e383eb13743c77a5cc67b4231f7248117da1be3233836cbd954cd1f368360795ec7457021370f1fb4633f1259c9898095583e31631f5f654e04fa9d0516713b40f21fa91a90436d813d1327d5cd115587fa0a5ecdceba8e0a6dcffc2bcb309a5686dd174e364ef439568642e606af50c803b874b74e24fd2d70454f71e0035f21d2a6990e4c097ba9870ff6942d5ad96317fd8ab1894004b66934073bfcdd4496b394c5fba1642b551a4299ccff04ba60c73e31f480b484785359"^^http://www.w3.org/2001/XMLSchema#hexBinary; #key @http://www.w3.org/ns/auth/cert#exponent "65537"^^http://www.w3.org/2001/XMLSchema#integer}
 
-finalKeyPGGraph: Rdf#Graph =  {#key @http://www.w3.org/ns/auth/cert#modulus "0087acb657366c6a911a0ce2470c24b8d85dd21ae76c6001db37d122c2eafe0fe2ae35541ccdd3c1c81603d98dd7d61b2c31c8605f81fbc3566604b7755793698836caaa99d868477c46b5b735529b50c0acfea6e10fc9b953697a67d57499801beb651e7e08343131a00d1873ab753ce5e79fc961874ee132472f7f210bc38966081fe263c620b67469b52cc555a26dee4fa10d8f40959e0e13516cc0bd1c1669ce53367d28248149142429b127f01e13d9bf21de52a4ac5694d5038a94178e144823b152fa19cbbc094dd40ddbd41e2195a19081125887fccedae21c50660c629a6f2ba4c85a54e1a6472f90b7b62ac7fe58fdae9daa0edc8edec49802fbfac7"^^http://www.w3.org/2001/XMLSchema#hexBinary; #key @http://www.w3.org/ns/auth/cert#exponent "65537"^^http://www.w3.org/2001/XMLSchema#integer}
 ```
-
-As evident, the Blank node is now changed to something more readable and useful. 
+As evident, the pointer of this graph is a useful identifier "#key" and the graph itself is the key information.
```

Comment:
Why is it useful? Also note that it is a relative URI. That should help you understand what is interesting about it when you post it. (It is useful also because how it is used in access control settings).